GiEnJul/Dtos/CreateUserDto.cs

```diff
@@ -0,0 +1,17 @@
+﻿namespace GiEnJul.Dtos
+{
+    public class CreateUserDto
+    {
+        public string Email { get; set; }
+        public string Password { get; set; }
+        public string Location { get; set; }
+        public string Role { get; set; }
+        public string? Institution { get; set; }
```

Comment:
```suggestion
        public string Institution { get; set; }
```
We don't have the "nullable" feature enabled in this project so all strings are nullable

Reply:
okey 👍